examples/pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: 5.4
+lockfileVersion: 5.3
```

Comment:
This is being downgraded? Is your local pnpm old?

Reply:
probably. I'll fix it.